charts/opentelemetry-collector/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v2
 name: opentelemetry-collector
-version: 0.104.0
```

Comment:
I don't think we need to upgrade the chart's version for this documentation update.